server/services/netatmo/lib/update/netatmo.update.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable jsdoc/check-param-names */
+const { EVENTS, WEBSOCKET_MESSAGE_TYPES } = require('../../../../utils/constants');
+const { getDeviceFeatureBySelector } = require('../../../../utils/device');
+const { compareValue } = require('../utils/format');
+const logger = require('../../../../utils/logger');
+
+/**
+ * @description Poll value of a Netatmo devices
+ * @param {string} key - Data received.
+ * @param {Object} device - Data received.
+ * @param {string} deviceSelector - Data received.
+ * @param {string} featureType - Data received.
+ * @param {number|boolean} newValue - Data received.
+ * @example
+ * updateNHC();
+ */
+async function updateFeature(key, device, deviceSelector, featureType, newValue) {
+  try {
+    const featureTypeSelector = `${deviceSelector}-${featureType.replace(/_/gi, '-').toLowerCase()}`;
+    const feature = await getDeviceFeatureBySelector(device, featureTypeSelector);
+    if (compareValue(feature.last_value, newValue)) {
```

Comment:
As said earlier in the review, I don't think this is something we want at the netatmo level.